api/src/home/analysis-platform/data/ForecastDS/FoR-BP/Blueprints/Response.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
didn't quite understand why we have two equal Response.json blueprints.

seems to be duplicate of some other blueprints as well

(in FoR-BP/Blueprints and in ForApp/io)

Reply:
The first "NOTE" describes this poorly. It's simply the Stask that was provided too us references these blueprints in these datasources